### ISSUE TITLE
Bot can't get initial pokestops

### ISSUE THREAD

**Issue by mtchm** [jabbink/PokemonGoBot#752]
| Getting initial pokestops...
| got 0 pokestops

It's definitely not the location, I've been using this location for a long time now.

**Version:**
alpha4

**Operating System:**
W10 x64

**Java version:**
1.8+

Some sort of soft ban?

**Comment by riorush** [comment]
Funny you should mention that i have just started getting this error after using the bot flawlessly for about half an hour. Coincidence?

**Comment by mtchm** [comment]
Did you also restart it after it started blacklisting all the pokestops it encountered?

**Comment by riorush** [comment]
Tried restarting three times, im using the default config gps long / lat so i know there's no mistake there.

**Comment by mSmithe** [comment]
this message only appeared for me when i used a location with the nearest pokestop lies a few 10's of km away...

**Comment by mtchm** [comment]
I told you it's not the location, I've been using it for quite a while....

**Comment by mojin7** [comment]
I also have had this issue. With Alpha 3 and Alpha 4. I just had it happen. I ran the bot for about 90 minutes. Closed it. I cleaned up my items and pokemon by tranferring pokemon and trashing items. Then I closed the app. I killed the task on my phone. And went to restart the bot. This is how it has happened every time in that order. When I went home from work and started it up it worked fine.

**Comment by mojin7** [comment]
@mtchm Mine is also using the same location. I have also tried changing the location. Same issue.

**Comment by mojin7** [comment]
Update. While the bot is running I opened task manager, right clicked java.exe, and ended process tree. I ran the bot and it found pokestops. Coincidence? Maybe.
I tried restarting the PC, I tried just just ending the process. But after ending process tree while it was running fixed the issue.

**Comment by riorush** [comment]
@mojin7 - Did exactly that and it seemed to work. However given the timing and that it's now working for both of us I have a feeling there was something server side affecting the bot.

**Comment by mojin7** [comment]
Ill update again next time it happens. Within the next hour or too. Because it is either that it worked for both of us or we had the exact same server issue at the same time. Neither is that unlikely. 
For anybody reading this. I'm running windows 7. Java 1.8 101. Alpha4 bot.

**Comment by yashendra2797** [comment]
Server issues? It seems to resolve itself after some time.

**Comment by Casual-User** [comment]
I think it's a modified soft ban.
I get it every time when modifying my inner city coordinates to fast.

**Comment by mtchm** [comment]
@Casual-User I don't think so. The error started occuring while I had been botting for atleast 50 minutes. Plus as I've said before, I have been using this location for quite a while, no switching locations in the meantime.

**Comment by Casual-User** [comment]
There might be other reasons for getting this kind of pseudo ban. Like walking to fast or walking in a pattern that is more bot like than others. I also got it easier in NYC than in San Francisco.
Currently, the only work around that works for me is waiting for like 20min before restarting the bot.

**Comment by mtchm** [comment]
@Casual-User You're right. It's working again right now. Maybe it has something to do with not following streets. I'm currently using follow_streets=false. hbu?

**Comment by Cypherke** [comment]
had it too
lowered initial_map_size and retried, always works

**Comment by TheWayAway** [comment]
@Cypherke 

Is that a setting not in the default config?
`initial_map_size`

**Comment by Cypherke** [comment]
@TheWayAway oh lol sorry i m not on the develop branch atm :) ... well expect it if #728 gets through

**Comment by yashendra2797** [comment]
@mtchm I'm currently using `follow_streets=true`

**Comment by jabbink** [comment]
Please literally copy and paste the `latitude` and `longitude` from your config.

I suspect you switched those and are now swimming in the ocean (somehow many people do that; the default order for those two values everywhere is first latitude, THEN longitude (@riorush specifically said "long / lat" which is the non-default order of these values anywhere GPS related))

**Comment by mojin7** [comment]
@jabbink This is possible in some cases. But I personally, and a couple others in this thread, are having this issue after making no changes to a working config.

**Comment by KyleBoyer** [comment]
I also have this issue with the latest build. However, I also have a previous build. Both are using the same coordinates. The old build finds 3 pokestops right away, the new one isn't finding any. I literally set my GPS location on top of a pokestop. I'm trying a reboot right now, I'm on Mac OS X.

**Comment by mojin7** [comment]
I think that the several "fixes" here may be coincidence . For me at least, this happens only occasionally. And without any changes. Last night and today I have had no issues starting the bot at least a half a dozen times. But when it does happen, waiting a little while resolves it. Whether it is a few minutes trying to start it over and over or trying to kill the process fully, or a few minutes altering map size etc., it seems like the "few minutes" is the constant here.

**Comment by mtchm** [comment]
I don't know how to reproduce it but I haven't had the issue in over 20 hours. 

@jabbink It doesn't have anything to do with my latitude/longitute settings since the error started to occur after I had been running the bot for an hour or so. 

The only reliable fix for me was waiting for about 15 minutes and restarting the bot.

**Comment by prestonvanloon** [comment]
I am having this issue right now
```
latitude=32.7833333
longitude=-96.8
```

**Comment by prestonvanloon** [comment]
Tried again with a different location:
```
latitude=32.960600
longitude=-96.828001
```
And got this:
```
03 Aug 15:26:41 (default): Getting map of (32.960591987337814, -96.82795559695778)
03 Aug 15:26:41 (default): Getting initial pokestops...
03 Aug 15:26:42 (default): Got 0 pokestops
03 Aug 15:26:42 (default): Retrying in 10 seconds...
```

**Comment by pizdece** [comment]
Lowered initial_map_size from 9, to 6 - works well. Thanks @KyleBoyer !
